packages/scim_pickens/docs/README.md

```diff
@@ -0,0 +1,3 @@
+# SCIM Pickens 🤠
+
+SCIM Pickens is a [SCIM](https://www.simplecloud.info/) library. TODO: fill this out with a better description
```

Comment:
A client library for use in Nitro? Should that really be part of the generic power-tools, and not part of NitroID's suite of client code?